src/components/buttons/DangerActionButton.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import Button from './Base'
+import styled from 'styled-components'
+
+const StyledButton = styled(Button)`
```

Comment:
Change name for importanWarning or something similar. Danger does not seem like a good work to exist inside the code